lib/helpers/defaultTopicTrigger.js

```diff
@@ -12,8 +12,11 @@ const config = require('../../config/lib/helpers/defaultTopicTrigger');
 function fetchAll() {
   // TODO: Loop through all pages of query results - currently returns 100.
   return contentful.fetchByContentTypes(config.defaultTopicTriggerContentTypes)
-    .then(contentfulEntries => contentfulEntries.map(module.exports
-      .parseDefaultTopicTriggerFromContentfulEntry));
+    .then(contentfulEntries => Promise.all(contentfulEntries.map(module.exports
```

Comment:
Minor, but we could make use of the [`Promise.map`](http://bluebirdjs.com/docs/api/promise.map.html) function here to simplify this a little more.